.github/workflows/ci.yml

```diff
@@ -27,8 +27,8 @@ jobs:
         run: ./gradlew sonarqube
   integration-test:
     runs-on: ubuntu-latest
-    if: ${{ startsWith(github.ref, 'refs/heads/master') || startsWith(github.base_ref, 'master') || startswith(github.ref, 'refs/heads/release') }}
```

Comment:
Please revert this, failing `master` is enough?

Reply:
shouldn't the integration test run for all branches before merging?